Standardize App ports

n8eloy opened the pull request:
## Closes #164 

Altered ports for every application following the standard guidelines as listed in the issue mentioned above. Documentation was updated as well.

Krlier commented:
Thanks for this, @n8eloy! But we also need to change some of the images that shows the port number to the one we're currently using.

n8eloy commented:
Hey @Krlier ! I've finished working on changing the images. Please give me a hint if anything is missing.

n8eloy commented:
Hey @Krlier ! I am glad to help! I've fixed that image recently (last commit). I guess it's showing like that due to browser caching.
![Screenshot from 2019-10-17 10-50-15](https://user-images.githubusercontent.com/31492173/67014878-f6229680-f0cb-11e9-9992-87615474063b.png)

Krlier commented on owasp-top10-2017-apps/a6/misconfig-wordpress/Makefile:
```diff
@@ -8,7 +8,7 @@ COLOR_GREEN = \033[32m
 
 PROJECT := A6 - Vulnerable Wordpress Misconfig
-PORT := 8000
+PORT := 10006
```
Changing the A6 - Misconfig-Wordpress port breaks the app because of the WordPress inside, as its URL is actually pointing to `http://localhost:8000`.

I suggest leaving this app out, for now.

n8eloy replied:
Shall I revert all changes made to A6 ports, then?

Krlier commented on owasp-top10-2017-apps/a8/amarelo-designs/deployments/docker-compose.yml:
```diff
@@ -8,7 +8,7 @@ services:
       - "../app/:/app"
     ports:
-      - "5000:5000"
+      - "10008:10008"
```
Flask in this app runs on port `5000` inside the container, so we need to forward it to `10008` outside to access it.

Do you mind changing this line to `10008:5000`?

n8eloy commented on owasp-top10-2017-apps/a4/vinijr-blog/deployments/docker-compose.yml:
```diff
@@ -8,4 +8,4 @@ services:
       - "../app/:/var/www/html"
     ports:
-      - 10080:80
+      - 10004:10004
```
Fixed!

n8eloy commented:
Hey @Krlier ! I've reverted accidental changes made to Docker Container ports mentioned. If you find more of those, please let me know. Also, I've moved A6 Misconfig-Wordpress back to it's original state for now.

Krlier commented:
> Hey @Krlier ! I've reverted accidental changes made to Docker Container ports mentioned. If you find more of those, please let me know. Also, I've moved A6 Misconfig-Wordpress back to it's original state for now.

Thanks so much for contributing,@n8eloy! Looks great! 🎊🎉
